bundles/automation/org.eclipse.smarthome.automation.module.script.rulesupport/src/main/java/org/eclipse/smarthome/automation/module/script/loader/internal/file/ScriptFileProvider.java

```diff
@@ -0,0 +1,217 @@
+/**
```

Comment:
> You also told me, I do not have to rename the namespaces ;)

This can only be some misunderstanding - clearly, any class within a bundle should use the namespace of its bundle. You probably refer to my statement "You can easily keep all the stuff in a separate java package that is independent of the rest of rulesupport." - but this didn't say that the namespace of this package should not follow the guidelines. As it is anyhow an internal package, it can easily be renamed once it is decided to a different bundle.